homeassistant/components/barry/strings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "error": {
+      "invalid_access_token": "[%key:common::config_flow::error::invalid_access_token%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "access_token": "[%key:common::config_flow::data::access_token%]"
+        },
+        "description": "Enter your access token from the authentication section in https://barry.energy/dk/faq/360004776259/360016952420",
+        "title": "Barry"
+      },
+      "metering_point": {
+        "data": {
+          "metering_point": "[%key:common::config_flow::data::metering_point%]"
+        },
+        "description": "Select one metering id to get data",
```

Comment:
What if users want to use more metering id's? Is that (currently) not supported, since they also can only add one single Barry integration.

Reply:
Thanks for the concern but In the initial release of our brand in Home Assistant, we are supporting one metering point per customer. And most of our customers have a single metering point at the moment. However, We are planning to support multiple meters per customer in the next version.